src/index.css

```diff
@@ -280,3 +288,56 @@ input[type="search"] {
   padding-top: 1rem;
   padding-left: 0.5rem;
 }
+
```

Comment:
Everything below is removed for now, because the React-Aria Switch is no longer used.